pkg/synopsysctl/cmd_create.go

```diff
@@ -55,29 +55,7 @@ var mockAlert bool
 var createCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create a Synopsys Resource in your cluster",
-	//(PassCmd)Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
This code is for passing values to kubectl/oc. Yesterday we decided to leave the code but just disable it so that it's there if we want it in the future. It is in git history so I think it's okay if we delete it. What do you think?